### DISCUSSION TITLE
lambda emacs in action short video

### DISCUSSION THREAD

**Discussion by ghost** [Lambda-Emacs/lambda-emacs#16] [General]
https://vimeo.com/manage/videos/735622551

I updated the packages with auto-update-packages.el in config.el
still kind-icon error not resolved

**Comment by mclearc** [comment]
I suspect the problem is that `corfu-margin-formatters` is not defined because in the splash screen that you have it hasn't been loaded yet. Have you tried just requiring corfu at startup and seeing if you still get the error?

**Comment by ghost** [comment]
is not defined because in the splash screen ? you mean dashboard ?
we have to enable one or the other ? between splash or dashboard ?

I require corfu in config.el ?

**Reply by mclearc** [reply]
I think the problem is that right now `kind-icon` is deferred for one second, which may be a problem with dashboard (I don't know -- since I don't use dashboard). So yeah -- easiest thing to do is either require it from your config or go into `lem-setup-completion` and change `:defer 1` to `demand t`

**Reply by mclearc** [reply]
Having just tested -- I don't see a problem using dashboard instead of the `lem-setup-splash` module.

**Comment by ghost** [comment]
Time permitting, can you make a dynamic dashboard ? 
clickable icons like the icons on the splash, with a nice png image perhaps every user can choose between many png images

**Reply by mclearc** [reply]
`dashboard.el` already seems to cover that. Were you able to resolve the error? Otherwise I can test out `dashboard.el` and see what the issue is.

**Reply by ghost** [reply]
I only changed this line https://github.com/wereket/lambda-emacs/blob/main/.local/lambda-library/lambda-setup/lem-setup-dashboard.el#L29
this is how it look when I start lambda emacs
https://vimeo.com/manage/videos/735736151

**Comment by ghost** [comment]
I updated the issue on github with a recent video

**Reply by mclearc** [reply]
Yes - I'll take a look at your setup and see if I can figure out what's going on

**Reply by mclearc** [reply]
Hi @wereket -- I pushed a change to lambda-emacs that you can either pull or you can go into the setup files for `lem-setup-completion` and change [this line](https://github.com/Lambda-Emacs/lambda-emacs/commit/4d907ec72fc21859f9321b2e81476e9a7a3f56b0). That should fix it.

**Reply by mclearc** [reply]
In general I would encourage you to put all your personal configuration and packages in you `config.el` file and in user directory more generally. ideally the lem-setup files don't need to be touched.
